bvm/ballerina-core/src/main/java/org/ballerinalang/model/util/serializer/BValueSerializer.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * you may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.ballerinalang.model.util.serializer;
+
+import org.ballerinalang.model.values.BValue;
+
+/**
+ * Convert Java object into {@link BValue} representation.
+ *
+ * @since 0.98.1
```

Comment:
Check the version. It should be the next major release version AFAIK. (aka. 0.982.0)